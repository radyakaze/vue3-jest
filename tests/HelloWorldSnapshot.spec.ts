import { mount } from "@vue/test-utils";
import HelloWorld from "../src/components/HelloWorld.vue";


describe("Hello World Component Snapshot", () => {
  const msg = "Hello Jest";
  const wrapper = mount(HelloWorld, {
    props: {
      msg,
    },
  });

  it("should match snapshot", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

  it("should match inline snapshot", () => {
    expect(wrapper.find('h1').text()).toMatchInlineSnapshot('"Hello Jest"');
  });
});
