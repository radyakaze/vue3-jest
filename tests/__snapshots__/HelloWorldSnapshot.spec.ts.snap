// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello World Component Snapshot should match snapshot 1`] = `
"<h1>Hello Jest</h1>
<div class="card"><button type="button">count is 0</button>
  <p> Edit <code>components/HelloWorld.vue</code> to test HMR </p>
</div>
<p> Check out <a href="https://vuejs.org/guide/quick-start.html#local" target="_blank">create-vue</a>, the official Vue + Vite starter </p>
<p> Install <a href="https://github.com/vuejs/language-tools" target="_blank">Volar</a> in your IDE for a better DX </p>
<p class="read-the-docs">Click on the Vite and Vue logos to learn more</p>"
`;
